### PR TITLE
Pro 6 task details view navbar

### PR DESCRIPTION
**Pull request by dimoiuBogdan**

### CODE REVIEW

**Review by StefanSG13 — COMMENTED** [review]
[`### _**I cannot say for sure if this is the right approach**_`](cornhub.com)

File: package.json
Comment (StefanSG13) [inline package.json:1]: 💪🏾💪🏾💪🏾💪🏾💪🏾💪🏾💪🏾

**Review by FratilaAndrei — CHANGES_REQUESTED** [review]
Please fix what i told ya

**Review by FratilaAndrei — APPROVED** [review]
Good job this time

**Pull request merged**